jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/ui/utils/ToolbarManager.kt

```diff
@@ -0,0 +1,53 @@
+package com.e_conomic.weatherapp.ui.utils
+
+import android.support.v7.graphics.drawable.DrawerArrowDrawable
+import android.support.v7.widget.RecyclerView
+import android.support.v7.widget.Toolbar
+import com.e_conomic.weatherapp.App
+import com.e_conomic.weatherapp.R
+import com.e_conomic.weatherapp.extensions.slideEnter
+import com.e_conomic.weatherapp.extensions.slideExit
+import com.e_conomic.weatherapp.ui.activities.SettingsActivity
+import org.jetbrains.anko.startActivity
+import org.jetbrains.anko.toast
+
+interface ToolbarManager {
```

Comment:
IMO this should be transformed into an `abstract class`. Even though you can have implemented methods in Kotlin interfaces, I think this is a wrong approach when considering that we need to initialise the manager in some way..